zerver/lib/event_queue.py

```diff
@@ -453,6 +493,7 @@ def setup_event_queue():
     send_restart_events(immediate=settings.DEVELOPMENT)
 
 def fetch_events(query):
+    # type: (Dict[str, Any]) -> Dict[str, Any]
```

Comment:
I don't think `query` should be `Mapping` instead of `MutableMapping`. As far as I can see, we're not modifying `query` in this function.